crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -741,11 +744,16 @@ fn process_room_properties(
     if let Some(recency_timestamp) = &room_data.timestamp {
         room_info.update_recency_timestamp(*recency_timestamp);
 
-        changes
-            .room_info_notable_updates
-            .entry(room_id.to_owned())
-            .or_default()
-            .insert(RoomInfoNotableUpdateReasons::RECENCY_TIMESTAMP);
+        // If it's not a new room, let's emit a `RECENCY_TIMESTAMP` update.
+        // For a new room, the room will appear as new, so we don't care about this
+        // update.
+        if is_a_new_room.not() {
```

Comment:
nit: i know you love `.not()`, but how about using a plain `!` here :)

Reply:
Done. I really dislike `!`. My fight isn't over 😉.